### ISSUE TITLE
Some -f | --framework examples are invalid

### ISSUE THREAD

**Issue by leecow** [dotnet/docs#5268]
https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-new?tabs=netcore2x 

For example, `dotnet new mvc -au None -f netcoreapp2.0` and `dotnet new xunit --framework netcoreapp2.0` both error with

```
Error: Invalid parameter(s):
-f netcoreapp2.0
    'netcoreapp2.0' is not a valid value for -f (Framework).
```

**Comment by mairaw** [comment]
Thanks @leecow. This is due to changes in .NET Core 2.1.300 SDK that no longer accepts -f. We're just starting to cover the 2.1 changes now.

**Comment by mairaw** [comment]
Fixed with PR #5931. However, I've filed a new issue to cover the scenario when the option does work when you install old templates.
